lib/IDE/CodeCompletionDiagnostics.cpp

```diff
@@ -157,31 +157,3 @@ bool swift::ide::getContextFreeCompletionDiagnostics(
   }
   return true;
 }
-
-bool swift::ide::getContextualCompletionDiagnostics(
```

Comment:
I'm not a fan of this mismatched implementation between context-fee and contextual.
What do we want to avoid here? Couldn't `getContextualCompletionDiagnostics()` receive ASTContext and NameForDiagnostics?